controller/experimentController.js

```diff
@@ -0,0 +1,89 @@
+const MongoStorage = require('../db/MongoStorage');
+const {json} = require("express");
+const experimentDB = new MongoStorage("experiment");
+
+const getAllExperiments = async (req, res) => {
+    const result = await experimentDB.find();
+    res.status(200).json({ result });
+}
+
+const getExperimentById = async (req, res) => {
+    if (req.params.experiment_id) {
+        const experiment_id = req.params.experiment_id;
+        const result = await experimentDB.retrieve(experiment_id);
+        res.status(200).json({ result });
+    }
+}
+
+const getExperimentsByAccountId = async (req, res) => {
+    if (req.params.account_id) {
+        const account_id = req.params.account_id;
+        const experiments = await experimentDB.findByAccount("account_id", account_id);
+        res.status(200).json( {experiments});
+    }
```

Comment:
Same. Here you need to add some checks:
1. is req.params passed
2. is req.params.account id valid
3. is experiments valid
throw the fitting error